### PR TITLE
Add blog example

### PR DESCRIPTION
**Pull request by juanbzz**
This is a very simple blog example built with Next.js with the minimal features users would need. Including the `now.json` file to deploy to Now.

- [Example deployed in Now](https://nextjs-blog-starter.now.sh/)
- [Personal blog in production using the example](https://jolvera.dev)

### CODE REVIEW

File: examples/blog-starter/posts/get-blog-posts.js
Comment (lfades) [inline examples/blog-starter/posts/get-blog-posts.js]: @j0lv3r4 this is not necessary if you can import the file and get the meta from it, like [here](https://github.com/zeit/next-site/blob/master/www/scripts/build-rss.js#L8), because you're adding the file to the entries of webpack, you can use everything that's allowed by a page, even mdx.

File: examples/blog-starter/posts/index.js
Comment (lfades) [inline examples/blog-starter/posts/index.js]: Is this file required for something ? `get-blog-posts` will be handled in build time anyway

File: examples/blog-starter/static/feed.json
Comment (lfades) [inline examples/blog-starter/static/feed.json]: What's this file for ? looks like the output of `rss-feed.js`, but the feed is automatically generated so I don't think we need it here for the example

File: examples/blog-starter/now.json
Comment (Timer) [inline examples/blog-starter/now.json]: Why is this route present?

File: examples/blog-starter/static/feed.json
Comment (j0lv3r4) [reply examples/blog-starter/static/feed.json]: Yes. This is a leftover. I'll remove it.

File: examples/blog-starter/now.json
Comment (j0lv3r4) [reply examples/blog-starter/now.json]: I think I pulled it from another project `now.json` file. I assume it is redundant to have it here, if that's the case I'll remove it.

File: examples/blog-starter/posts/get-blog-posts.js
Comment (j0lv3r4) [reply examples/blog-starter/posts/get-blog-posts.js]: As we discussed, we'll leave it as it is for now. Using `readFileSync` is better in this case.

**Review by lfades — APPROVED** [review]
Thank you @j0lv3r4 is working perfectly now 👌 🐐 💯

**Comment by lfades** [comment]
@j0lv3r4 The linter is complaining about trailing spaces in the [build](https://circleci.com/gh/zeit/next.js/9267?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), can you fix those ? I'll merge right after

**Comment by j0lv3r4** [comment]
@lfades just did!

**Comment by iamstarkov** [comment]
nice

**Pull request merged**
